### PR TITLE
Add few more facts to satask

### PR DESCRIPTION
**Pull request by debugger22**
Add following facts:
- neg + neg -> neg
- pos + pos -> ~neg
- neg + neg -> ~pos

@asmeurer I think, we can create a general system of handling inequalities in assumptions by using expression tree. I'll work on that later. For now I've added missing facts.

### CODE REVIEW

File: sympy/assumptions/sathandlers.py [hunk 1]
```diff
@@ -300,6 +300,9 @@ def register_fact(klass, fact, registry=fact_registry):
     (Mul, Equivalent(Q.zero, AnyArgs(Q.zero))),
     (MatMul, Implies(AllArgs(Q.square), Equivalent(Q.invertible, AllArgs(Q.invertible)))),
     (Add, Implies(AllArgs(Q.positive), Q.positive)),
+    (Add, Implies(AllArgs(Q.positive), ~Q.negative)),
```
Comment (jksuom) [inline sympy/assumptions/sathandlers.py]: We could have ``Implies(Q.positive, ~Q.negative)``. Or perhaps ``Implies(Q.positive, Q.nonnegative)`` and ``Implies(Q.nonnegative, ~Q.negative)``. Then this line would follow from the line above.

Comment (asmeurer) [inline sympy/assumptions/sathandlers.py]: Those facts are true for every object. They should go in the known_facts.

File: sympy/assumptions/sathandlers.py [hunk 2]
```diff
@@ -300,6 +300,9 @@ def register_fact(klass, fact, registry=fact_registry):
...
+    (Add, Implies(AllArgs(Q.negative), Q.negative)),
+    (Add, Implies(AllArgs(Q.negative), ~Q.positive)),
```
Comment (asmeurer) [inline sympy/assumptions/sathandlers.py]: This fact also ought to be redundant. The assumptions should already know that negative >> ~positive.

File: sympy/assumptions/sathandlers.py (hunk 1 above)
Comment (debugger22) [inline sympy/assumptions/sathandlers.py]: For some reason it doesn't work when I add it in the `known_facts`.

Comment (asmeurer) [inline sympy/assumptions/sathandlers.py]: We should investigate that. Can you push a commit with it?

**Comment by debugger22** [comment]
It works now.

File: sympy/assumptions/ask.py [hunk 3]
```diff
@@ -378,6 +378,8 @@ def get_known_facts():
         Implies(Q.antihermitian, ~Q.hermitian),
         Equivalent(Q.negative, Q.nonzero & ~Q.positive),
         Equivalent(Q.positive, Q.nonzero & ~Q.negative),
+        Implies(Q.positive, ~Q.negative),
+        Implies(Q.negative, ~Q.positive),
```
Comment (asmeurer) [inline sympy/assumptions/ask.py]: These two facts are redundant. The one is the contrapositive of the other.

Comment (asmeurer) [inline sympy/assumptions/ask.py]: You should add a test for this fact directly.

Comment (debugger22) [inline sympy/assumptions/ask.py]: Right. Done.

**Pull request merged**